<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddpatientDlg</class>
 <widget class="QDialog" name="AddpatientDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>480</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Gender">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Age">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Phone">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>联系电话</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Phone">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Age">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>就诊人姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Year">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>岁</string>
   </property>
  </widget>
  <widget class="QLabel" name="labe_AddInformation">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Modern No. 20</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请添加新就诊人信息：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Male">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetPatient">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>711</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Female">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>340</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DeletePatient">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>470</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>删除就诊人</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>69</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Patient.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddpatientDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddpatientDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
